src/applications/proxy-rewrite/sass/consolidated-patches.scss

```diff
@@ -128,6 +128,12 @@ header.merger {
     }
   }
 
+  @include media($medium-screen) {
+    .vetnav-panel .mm-link-container .vetnav-level2 {
+      line-height: 1em;
+    }
+  }
+
```

Comment:
This whole file is basically a collection of hacks. Turns out injecting an entire navigation into a page we don't control  introduces weird edge cases, which this file addresses. The idea is that this will be removed once Web Brand Consolidation has progressed further.